tests/mbt/model/libraries/extraSpells.qnt

```diff
@@ -139,6 +139,71 @@ module extraSpells {
         __set.fold(List(), (__l, __e) => __l.append(__e))
     }
 
+  /// The type of orderings between comparable things
```

Comment:
Why the `///` here and later instead of `//`?

Reply:
Good question, this is copy-pasted from the Quint "standard library" (there is no actual standard library yet, so copy-pasting what you need is atm the canonical way of using it) and their style differs sometimes

Reply:
I see. I guess in this case we could adhere to our own standard with just `//`.